corehq/apps/app_manager/views/releases.py

```diff
@@ -153,6 +160,8 @@ def save_copy(request, domain, app_id):
     track_built_app_on_hubspot.delay(request.couch_user)
     comment = request.POST.get('comment')
     app = get_app(domain, app_id)
+    if not app.is_remote_app():
+        app.update_mm_map()
```

Comment:
Does this have to be done before `validate_app`? Otherwise I might suggest putting it in `app.make_build`